integrationtests/python_src/px4_it/mavros/missions/VTOL_mission_1.plan

```diff
@@ -10,71 +15,74 @@
         "items": [
             {
                 "AMSLAltAboveTerrain": null,
-                "Altitude": 12,
+                "Altitude": 15,
                 "AltitudeMode": 0,
                 "autoContinue": true,
-                "command": 16,
+                "command": 84,
                 "doJumpId": 1,
                 "frame": 3,
                 "params": [
                     15,
                     0,
                     0,
                     null,
-                    47.398157669127094,
-                    8.546033631711538,
-                    12
+                    47.397731862593744,
+                    8.543269295853861,
+                    15
                 ],
                 "type": "SimpleItem"
             },
             {
+                "AMSLAltAboveTerrain": null,
+                "Altitude": 50,
```

Comment:
All of the VTOL missions are failing because the altitude is too high. I tested locally and after 60 sec it's only down to 15m. We either need to lower the altitude or increase the timeout.